### PR TITLE
Frontend - Buttons components

### PR DESCRIPTION
**Pull request by jrgarou**
**Buttons**
- buttons/BaseButton.js
 The base for 3 buttons(Anchor, Button, NavLink) with hover and colors scheme `<... white>``<... purple>`` <...>`(gray)

- buttons/Anchor.js
  `<Anchor any<a>props>` `<Anchor underline>`

- buttons/Button.js
  `<Button/>`

- buttons/NavLink.js
  `<NavLink to='/uploader/upload-file' purple="true">Upload video</NavLink>`

### CODE REVIEW

**Comment by jellegerbrandy** [comment]
Hey @bent0b0x , this PR fails with linter errors that are, to me, ununderstandable. 

```src/scripts/components/foundations/buttons/BaseButton.js
  4:1   ✖  Expected indentation of 2  indentation                              
           spaces
```
line 4 of that file starts with 2 spaces.

File: src/scripts/components/foundations/buttons/Button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { css } from 'styled-components'
+import BaseButton from './BaseButton'
+
+//
```
Comment (bent0b0x) [inline src/scripts/components/foundations/buttons/Button.js]: @jrgarou why do these `//` keep getting added? 😄

File: src/scripts/components/structures/MainHeader.js [hunk 2]
```diff
@@ -41,6 +41,8 @@ const HeaderButtons = styled.div`
   justify-content: flex-end;
 `
 
+// foundation/widgets(move?)
```
Comment (bent0b0x) [inline src/scripts/components/structures/MainHeader.js]: @jrgarou can we delete this?

Comment (jrgarou) [reply src/scripts/components/structures/MainHeader.js]: Why remove this @bent0b0x ?

Comment (bent0b0x) [reply src/scripts/components/structures/MainHeader.js]: Because it is commented out. What is its purpose?

**Pull request merged**